packages/@lightningjs/ui-components/src/components/CardRadio/CardRadio.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import CardTitle from '../Card/CardTitle';
+import TextBox from '../TextBox';
+import * as styles from './CardRadio.styles.js';
+import Radio from '../Radio';
+
+export default class CardRadio extends CardTitle {
+  static get __componentName() {
+    return 'CardRadio';
+  }
+
+  static get __themeStyle() {
+    return styles;
+  }
+
+  static get properties() {
+    return [...super.properties, 'subtitle'];
+  }
+
+  static get tags() {
+    return [...super.tags, 'Subtitle', 'Radio'];
+  }
+
+  static _template() {
+    return {
+      ...super._template(),
+      Subtitle: {
+        type: TextBox,
+        signals: {
+          textBoxChanged: '_update'
+        }
+      }
+    };
+  }
+
+  _update() {
+    super._update();
+    this._updateSubtitle();
+    this._updateRadio();
+    this._updateDescriptionPosition();
+    this._updateSubtitlePosition();
+  }
+
+  _updateTitle() {
+    super._updateTitle();
+    this._Title.patch({
+      signals: {
+        textBoxChanged: '_update'
```

Comment:
Is this necessary? The Title textbox already has this set up in the Card template

```js
      Title: {
        type: TextBox,
        signals: {
          textBoxChanged: 'queueRequestUpdate'
        }
      }
```